playbooks/spark-integration-test-kubeadm-k8s/run.yaml

```
@@ -1,6 +1,7 @@
 - hosts: all
   become: yes
   roles:
+    - install-openjdk-8-jdk
```

```suggestion
    - install-openjdk
```

Please add option to specify the jdk version for this new role.